src/cmd-build

```
@@ -106,22 +106,14 @@ sha256sum_str() {
     sha256sum | cut -f 1 -d ' '
 }
 
-# Calculate kickstart checksum now and gather previous image build variables if any
-image_config_path="${configdir:?}"/image.yaml
-image_config=true
-if [ -f "${image_config_path}" ]; then
-    image_input="${image_config_path}"
-else
-    ks_path="${configdir:?}"/image.ks
-    if [ -f "${ks_path}" ]; then
-        echo 1>&2 "WARNING: Kickstart support is deprecated; migrate to image.yaml"
-        echo "Sleeping for 10 seconds."
-        sleep 10
-        image_input="${configdir:?}"/image.ks
-        image_config=false
-    else
-        fatal "Failed to find image.yaml in configdir"
-    fi
+# Calculate image input checksum now and gather previous image build variables if any
+ks_path="${configdir:?}"/image.ks
+if [ -f "${ks_path}" ]; then
+    fatal "Kickstart support was removed; migrate to image.yaml"
```

good idea to fail hard here!